### PR TITLE
Add case studies section to menu dropdown

### LINKED ISSUE #208: Case study index addition to product menu dropdown
Add new case studies section on menu dropdown

Figma: https://www.figma.com/file/et2KLDMVOPb0xeAmXFJXok/checklyhq.com?node-id=942%3A5735

### PR DESCRIPTION
**Pull request by pilimartinez**
Resolves #208 

- [x] Add case studies section to menu dropdown

### CODE REVIEW

File: src/scss/checkly-webapp-src/style/scss/components/_dropdowns.scss [hunk 1]
```diff
@@ -132,6 +131,7 @@
       }
       .right-side {
         width: 100%;
+        padding-top: 0px;
```
Comment (ianaya89) [inline src/scss/checkly-webapp-src/style/scss/components/_dropdowns.scss:134]: This could be just `0`

File: site/layouts/partials/navbar.html [hunk 2]
```diff
@@ -61,6 +61,18 @@
...
               </div>
               <div class="right-side">
+                <a class="dropdown-item right-side-item monitoring" href="/case-studies/">
+                  <div class="d-flex">
+                    <div class="menu-icon">
+                      <img src="/images/icons/case.svg" alt="case" width="32" height="32" />
+                    </div>
+                    <div>
+                      <div class="menu-title">Case studies</div>
+                      <div class="menu-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```
Comment (ianaya89) [inline site/layouts/partials/navbar.html:71]: `lorem` here, do we have the final version? cc @orcunilbeyli

**Review by ianaya89 — CHANGES_REQUESTED** [review]
Looks good, just a minor change.

File: src/scss/checkly-webapp-src/style/scss/components/_dropdowns.scss (hunk 1 above)
Comment (pilimartinez) [reply src/scss/checkly-webapp-src/style/scss/components/_dropdowns.scss]: fixed :)

**Comment by ianaya89** [comment]
@orcunilbeyli what should we do with this?

**Pull request closed without merging**